src/Http/Controllers/Admin/LoginController.php

```diff
@@ -289,4 +293,43 @@ protected function credentials(Request $request): array
     {
         return array_merge($request->only($this->username(), 'password'), ['published' => 1]);
     }
+
+    /**
+     * Try to autologin a user
```

Comment:
I feel the doc-blocks are a bit redundant here, the function names are descriptive enough here.

Reply:
Done